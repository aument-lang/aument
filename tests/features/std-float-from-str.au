print float::into("-12.5");