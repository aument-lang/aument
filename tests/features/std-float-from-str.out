float;-12.5